config/set/symfony52.php

```diff
@@ -21,4 +23,17 @@
                 new MethodCallRename('Symfony\Component\Mime\Address', 'fromString', 'create'),
             ]),
         ]]);
+
+    # https://github.com/symfony/symfony/blob/5.x/UPGRADE-5.2.md#security
```

Comment:
```suggestion
    // @see https://github.com/symfony/symfony/blob/5.x/UPGRADE-5.2.md#security
```

Reply:
We use `#` comment instead of `@see` in other Symfony-related sets like `symfony-43` so I went with it to keep the style the same.

Reply:
ok